src/common/constants/poStatuses.js

```diff
@@ -1,6 +1,17 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
 // eslint-disable-next-line import/prefer-default-export
 export const PO_WORKFLOW_STATUSES = {
```

Comment:
Maybe, order statuses could be used from https://github.com/folio-org/stripes-acq-components/blob/master/lib/constants/order.js#L4

Reply:
yes, it might be a good idea